docker/Dockerfile.exp

```diff
@@ -4,12 +4,15 @@ FROM esmvalgroup/esmvalcore:latest
 
 COPY ./environment.yml /src/ESMValTool/environment.yml
 WORKDIR /src/ESMValTool
-RUN conda update -n base -y conda && conda install -y -c conda-forge mamba && mamba env create --name esmvaltool --file environment.yml && conda clean --all -y
+RUN conda update -n base -y conda && conda install -y -n base -c conda-forge mamba
 
 # Make RUN commands use the new environment:
-SHELL ["conda", "run", "--name", "esmvaltool", "/bin/bash", "-c"]
+RUN echo "conda activate esmvaltool" >> ~/.bashrc
+SHELL ["/bin/bash", "--login", "-c"]
 
 COPY . /src/ESMValTool
+RUN mamba env update -n esmvaltool -f environment.yml
```

Comment:
Perhaps this can be re-integrated into an initial `mamba env create`?

Reply:
yeah, will restore that

Reply:
well that didn't work: `CondaValueError: prefix already exists: /opt/conda/envs/esmvaltool` - https://github.com/ESMValGroup/ESMValTool/runs/5375271463?check_suite_focus=true - *probably* from the ESMValCore container? I'm doing `update` and be done with it, it's sort of the same stuff anyway

Reply:
Ah, now there is your real real problem (see [here](https://github.com/ESMValGroup/ESMValTool/runs/5375271463?check_suite_focus=true#step:3:246)): Because the other container set the shell, everything here gets an extra double layer of `conda run`. Quoting the line I link above:
```
The command 'conda run --name esmvaltool /bin/bash -c mamba env create -n esmvaltool -f environment.yml' returned a non-zero code: 1
```

Reply:
yeah, that's why there already is an `esmvaltool` env and the shell needs to be reset via bashrc - man, this is a apinful container :laughing:

Reply:
OK went through this way - I think this is it, it all looks clean now